Vagrant/bootstrap.sh

```diff
@@ -85,6 +102,53 @@ apt-get install -y nodejs
 # Install Redis, ImageMagick, PhantomJS, and Libre Office
 apt-get install -y redis-server imagemagick phantomjs libreoffice
 
+# Install Solr
+# Fetch the Solr distribution and unpack the install script
+wget "$SOLR_MIRROR/$SOLR_DIST.tgz"
+tar xzf $SOLR_DIST.tgz $SOLR_DIST/bin/install_solr_service.sh --strip-components=2
+# Install and start the service using the install script
+bash ./install_solr_service.sh $SOLR_DIST.tgz -u $SOLR_USER -d $SOLR_MUTABLE -i $SOLR_INSTALL
+# Remove Solr distribution
+rm $SOLR_DIST.tgz
+# Stop Solr until we have created the core
+service solr stop
+
+# Install Tomcat and Fedora 4
+apt-get -y install tomcat7 tomcat7-admin
+usermod -a -G tomcat7 $installuser
+# Stop Tomcat until everything is installed
+service tomcat7 stop
+# Create Fedora roles (taken from fcrepo4-labs/fcrepo4-vagrant project)
+if ! grep -q "role rolename=\"fedoraAdmin\"" $TOMCAT_CONF/tomcat-users.xml ; then
+  sed -i '$i<role rolename="fedoraUser"/>
+  $i<role rolename="fedoraAdmin"/>
+  $i<role rolename="manager-gui"/>
+  $i<user username="testuser" password="password1" roles="fedoraUser"/>
+  $i<user username="adminuser" password="password2" roles="fedoraUser"/>
+  $i<user username="fedoraAdmin" password="fedoraAdmin" roles="fedoraAdmin"/>
+  $i<user username="fedora4" password="fedora4password" roles="manager-gui"/>' $TOMCAT_CONF/tomcat-users.xml
+fi
+if ! grep -q "$JDK_HOME" /etc/default/tomcat7 ; then
+  echo "JAVA_HOME=$JDK_HOME" >> /etc/default/tomcat7
+fi
+if ! grep -q 'fcrepo.home=' /etc/default/tomcat7 ; then
+  echo "JAVA_OPTS=\"${JAVA_OPTS} -Dfcrepo.home=$FCREPO4_DATA\"" >> /etc/default/tomcat7
+fi
+# Create Fedora data directory and make sure Tomcat 7 can write to it
+mkdir -p $FCREPO4_DATA
+chown ${FCREPO4_USER}:${FCREPO4_GROUP} $FCREPO4_DATA
+chmod 770 $FCREPO4_DATA
+# Build Fedora 4 from sources
+apt-get install -y git maven
+$RUN_AS_INSTALLUSER git clone --branch $FEDORA4_BRANCH --depth 1 $FEDORA4_REPO
```

Comment:
Good observation.  The Fedora 4 install instructions explicitly refer to the fcrepo4 directory, and so that's where I think that came from.  But, if, ultimately, we build from our own Fedora 4 repository, we might end up renaming it, so your improvement would be necessary.